Refactor Home Page, text API, Refactor Account Page (#3)
BlockLatticeOrg/website: README.md

# Website BlockLlaticeOrg

The Website Blockllattice is one of the parts of the [BlockLattice.org](https://blocklattice.org/) ecosystem. It's a Webpage that uses API of Block Accounts information. 

> If you want to contribute to this project, follow the steps bellow to install and code. But we recommend accessing our group on the telegran to chat better. You can also access the layout by giving pages in the zeplin tool.

Reviewer: ```suggestion
If you want to contribute to this project, follow the steps bellow to install and code. But we recommend accessing our group on the Telegram to chat better. You can also access the layout by giving pages in the zeplin tool.
```

> Telegran Group: https://t.me/joinchat/RWgbJxrrvxwsjfluRPhVoA

Reviewer: ```suggestion
Telegram group: https://t.me/joinchat/RWgbJxrrvxwsjfluRPhVoA
```

Layout Pages in Zeplin: https://app.zeplin.io/project/5f538cf5c70ac17ba984437f/dashboard

Explores API documentation: https://explorer.api.blocklattice.org/v1/docs

## Installation

You must have installed nodejs 10.13 or later. 

### Install dependencies

Enter project's shell, firt step is:
```bash
$ yarn
```

Second step to develop with Nextjs you will open the page on localhost:3000 in developer mode:
```bash
$ yarn dev
```

Before commit and push, please test your code using production simulation of Nextjs
```bash
$ yarn build && yarn start
```

> Do not forget create and push your new feature, after create Pull Request to Master

Reviewer: ```suggestion
Do not forget to create and push your new feature, then create a Pull Request to Master
```
